serving/fluentd/README.md

```diff
@@ -0,0 +1,40 @@
+# Fluentd Docker Image on Knative Serving
+
+Knative Serving uses a [Fluentd](https://www.fluentd.org/) docker image to collect
+logs. Operators can customize their own docker image and configuration to
+define logging output.
+
+## Requirements
+
+Knative requires the following Fluentd plugins to process log records:
```

Comment:
It's not clear from this whether these are already installed, or need to be installed.

Reply:
Also, can you explain why these are required? (vs simply that the logging will work better with these extensions)

Reply:
Done.